src/devices/machine/bl_handhelds_menucontrol.cpp

```diff
@@ -0,0 +1,214 @@
+// license:BSD-3-Clause
+// copyright-holders:David Haywood
+
+// Menu controller, stores / increments menu position etc. used on handhelds that typically have BL on the boot screen (BaoBaoLong?)
+
+#include "emu.h"
+#include "machine/bl_handhelds_menucontrol.h"
+
+DEFINE_DEVICE_TYPE(BL_HANDHELDS_MENUCONTROL, bl_handhelds_menucontrol_device, "BaoBaoLong", "BL Handhelds Menu Controller")
```

Comment:
That's supposed to be the device shortname, not a manufacturer field.

Reply:
brainfart moment that, changed things as requested